docs/docsgen/source/onnx_python/mapping.rst

```diff
@@ -0,0 +1,50 @@
+
+.. _l-mod-onnx-mapping:
+
+onnx.mapping
+============
+
+This module defines the correspondance between onnx numerical types
+and numpy numerical types. This information can be accessed
+through attribute :ref:`l-onnx-types-mapping` or through the functions 
+defined in :ref:`l-mod-onnx-helper`.
+
+.. contents::
+    :local:
+
+TensorDtypeMap
+++++++++++++++
+
+.. autoclass:: onnx.mapping.TensorDtypeMap
+
+.. _l-onnx-types-mapping:
+
+TENSOR_TYPE_MAP
++++++++++++++++
+
+.. exec_code::
+
+    import pprint
+    from onnx.mapping import TENSOR_TYPE_MAP
+
+    pprint.pprint(TENSOR_TYPE_MAP)
+
+Opset Version
```

Comment:
The part starting from this line seem out of place for this file. May be a copy-paste error? Feels like it should be in another file.

Reply:
True, I split the files and forgot to remove it.

Reply:
done